terraform/main.tf

```diff
@@ -134,6 +34,15 @@ module "py310_pandas_153" {
   layer_path     = "${path.root}/layers/python3.10/pandas_1.5.3/"
 }
 
+module "py310_pandas_214" {
```

Comment:
Can you put a comment above line 37 that this layer "py310_pandas_214" includes both pandas and geopandas?